temporal/fx.go

```diff
@@ -910,6 +910,14 @@ var TraceExportModule = fx.Options(
 		if err != nil {
 			return nil, err
 		}
+
+		exporterFromEnv, err := telemetry.EnvSpanExporter()
+		if err != nil {
+			return nil, err
+		} else if exporterFromEnv != nil {
+			exporters = append(exporters, exporterFromEnv)
+		}
```

Comment:
This is intentionally separate from the config-based span exporters, ie it's additive.